ToDoTime/ClientApp/src/shared/DynamicIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import loadable from "@loadable/component";
+import { IconBaseProps } from "react-icons/lib";
+
+interface DynamicIconProps {
+  nameIcon: string;
+  propsIcon?: IconBaseProps;
+}
+
+export const DynamicIcon = ({ nameIcon, propsIcon }: DynamicIconProps) => {
+  const lib = nameIcon
+    .replace(/([a-z0-9])([A-Z])/g, "$1 $2")
+    .split(" ")[0]
+    .toLocaleLowerCase();
+  const ElementIcon = loadable(() => import(`react-icons/${lib}/index.js`), {
+    resolveComponent: (el: JSX.Element) => el[nameIcon as keyof JSX.Element]
+  });
+
+  return <ElementIcon {...propsIcon} />;
```

Comment:
Same as previous comment, default icon and test for its usage.

Reply:
During the tests was noticed that the lazy import if the lib doesn't exist it will throw module not found and to catch the error and return a default component would require update the DynamicIcon component more than really necessary, since all our available icons for now just come from BoxIcons lib, so DynamicIcon was removed for now, if needed in the future can put it back and figure out how to load default imports.